pkg/handler/branch_test.go

```diff
@@ -37,3 +37,33 @@ func TestGetBranchByTag(t *testing.T) {
 		t.Errorf("handler returned unexpected body: got %v want %v", actualResponseBody, expectedResponseBody)
 	}
 }
+
+func TestGetActiveBranches(t *testing.T) {
+	req, err := http.NewRequest("GET", "/v1/aryamanrishabh/metricsjs/branch/getActiveBranches?unit=h&number=300", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	r := mux.NewRouter()
+	h := &branchHandler{}
+	r.HandleFunc("/v1/{owner}/{repo}/branch/getActiveBranches", h.GetActiveBranches).Methods("GET")
+
+	rr := httptest.NewRecorder()
+	r.ServeHTTP(rr, req)
+
+	if status := rr.Code; status != http.StatusOK {
+		t.Errorf("Handler returned incorrect status code")
+	}
+
+	expectedContentType := "application/json"
+	if contentType := rr.Header().Get("Content-Type"); contentType != expectedContentType {
+		t.Errorf("Handler returned incorrect content type")
+	}
+
+	expected := `{"branches":["master"]}`
+	got := strings.TrimSpace(rr.Body.String())
+
+	if got != expected {
+		t.Errorf("Handler returned unexpected body")
+	}
```

Comment:
nitpick: might be better to log expected and returned values to make debugging easier in the future

Reply:
in case of failures

Reply:
You are right, thank you for the review!